deploy/terraform/terraform-units/modules/sap_deployer/variables_local.tf

```
@@ -13,42 +51,49 @@ locals {
   // Default option(s):
   enable_secure_transfer = try(var.options.enable_secure_transfer, true)
 
+  // Resource group and location
+
+  region             = try(var.infrastructure.region, "")
+  landscape          = try(var.infrastructure.landscape, "global")
+  location_short     = try(var.region_mapping[local.region], "unkn")
+  vnet_mgmt_tempname = try(local.vnet_mgmt.name, "deployer")
+  prefix             = lower(try(var.infrastructure.resource_group.name, format("%s-%s-%s-infrastructure", local.landscape, local.location_short, local.vnet_mgmt_tempname)))
```

lower not apply for customer provided name :)

```suggestion
  prefix             =try(var.infrastructure.resource_group.name, lower(format("%s-%s-%s-infrastructure", local.landscape, local.location_short, local.vnet_mgmt_tempname)))
```

---

fixed